copy.go

```diff
@@ -87,7 +89,13 @@ func fcopy(src, dest string, info os.FileInfo, opt Options) (err error) {
 	}
 	defer fclose(s, &err)
 
-	if _, err = io.Copy(f, s); err != nil {
+	if opt.CopyBufferSize <= 0 {
+		_, err = io.Copy(f, s)
+	} else {
+		buf := make([]byte, opt.CopyBufferSize)
```

Comment:
Since `io.Copy` uses `copyBuffer` internally, I don't think using `if opt.CopyBufferSize <= 0` is a simple implementation.

Just for an idea: How about using default option with size `32 * 1024` used by [`io.Copy`](https://golang.org/src/io/io.go?s=14517:14534#L412) so we don't need to use `if` here.

Reply:
Perfectly fine. I'll change this.

Reply:
I changed this a little bit, but I need at least one `if` to check if the bufferSize was set appropriately

Reply:
I mean using `https://github.com/otiai10/copy/blob/main/options.go#L63` and `https://github.com/otiai10/copy/blob/main/copy.go#L205` to setup default buffer size, then we don't need to judge if it's set.
If the user intentionally set 0 or negative, let it error.
(or we can just prevent negative to use `uint` here)

Reply:
Wouldn't that mean I break everyones code who creates the `Options{` themself (without the `getDefaultOptions` function. When they upgrade to this version, their code breaks because this option would be 0 by default...

Reply:
Oh I see. Yes, I love that.

Reply:
Pushed that change. I didn't had a deep enough look into your lib to find out about this approach. Very cool. Thx